dumper/dataWriter.go

```diff
@@ -140,8 +139,14 @@ func GetRowsFromKeys(db *sql.DB, table schemareader.Table, keys []TableKey) [][]
 
 		values = append(values, "("+strings.Join(row, ",")+")")
 	}
-	sql := fmt.Sprintf(`SELECT %s FROM %s WHERE (%s) in (%s);`,
-		formattedColumns, table.Name, strings.Join(columnsFilter, ", "), strings.Join(values, ","))
+	// when columnsFilter is empty, do not append any where clause to prevent sql syntax error
+	// TODO: how it can happen to have no columnFilter when keys check at the beginning?
```

Comment:
Did you figure out this TODO? Should we remove the check for empty columns filters and let it return an error so we can analyze the case?